doc/whats_new.rst

```diff
@@ -107,6 +107,8 @@ Changelog
 
 Bug
 ~~~
+- Fix: bug with scaling of data in func:`mne.cov._compute_covariance_auto` by `David Sabbagh `_ 
```

Comment:
Missing line before the list.

```sh
/home/circleci/project/doc/whats_new.rst:110: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/home/circleci/project/doc/whats_new.rst:110: WARNING: Inline interpreted text or phrase reference start-string without end-string.
```

What I would do is wait until all CIs are done. (if green) I would add this empty line, commit with this message `[skip travis][skip av] remove whats_new emptyline warning` and push